hack/binaries

```diff
@@ -16,12 +16,12 @@ binariesLegacy() {
 	iidfile=$(mktemp -t docker-iidfile.XXXXXXXXXX)
 	copysrc="/usr/bin/."
 
-	case "$(echo "${TARGETPLATFORM}" | cut -d"/" -f1)" in
-		"darwin")
+	case "${TARGETPLATFORM}" in
+		darwin/*)
```

Comment:
`TARGETPLATFORM` can be just `darwin` as well. Eg. `docker build --platform darwin` works fine and missing sections are added automatically.

Reply:
oh! right 🤦‍♂️ should've seen that, I can fix that by changing the `case` to;

```bash
case "${TARGETPLATFORM%%/*}" in
```

Reply:
Fixed 👍